src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Map/GetElevationAtAPoint/GetElevationAtAPoint.xaml.cs

```diff
@@ -0,0 +1,107 @@
+// Copyright 2019 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.UI;
+using System;
+using System.Drawing;
+
+namespace ArcGISRuntime.WPF.Samples.GetElevationAtAPoint
+{
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "Get elevation at a point",
+        "Map",
+        "Retreive the elevation of a point on a surface",
+        "Tap anywhere on the surface to get the elevation at that point. Elevation is reported in meters since the scene view is in WGS84.")]
+    public partial class GetElevationAtAPoint
+    {
+        // URL of the elevation service - provides elevation component of the scene.
+        private readonly Uri _elevationUri = new Uri("http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer");
+
+        // Starting point of the observer.
+        private readonly MapPoint _observerPoint = new MapPoint(83.9, 28.42, SpatialReferences.Wgs84);
+
+        // Create the graphics overlay.d
```

Comment:
small typo

Reply:
Also, these aren't really created, I'd just say 'Graphics overlay for displaying points' or something similar. Same applies for comments below